SUMMARY.md

```diff
@@ -0,0 +1,19 @@
+## Table of Contents
+
+* [Ternary Operator](content/TernaryOperator/TernaryOperator.q.md) ** - minutes**
+    > In this lesson, you will learn how to simplify the condition of if/else statements using ternary operator. You will also learn the advantages of using ternary operator as an expression.
```

Comment:
Please rephrase this description here to remove redundant phrase `You will learn...` as much as possible we should use one `You will learn` phrase in each description. I think it is better to define the ternary operator first then, in the second statement we can use `You will learn`... 

> The ternary operator is... In this lesson, you will learn...